### PR TITLE
rework build list order for oc status

### PR DESCRIPTION
**Pull request by gabemontero**
Fixes https://github.com/openshift/origin/issues/6421

@deads2k PTAL

@bparees FYI

### CODE REVIEW

**Comment by deads2k** [comment]
@openshift/cli-review

**Comment by juanvallejo** [comment]
[test]

File: pkg/cmd/cli/describe/projectstatus.go [hunk 1]
```diff
@@ -841,12 +841,37 @@ func describeAdditionalBuildDetail(build *buildgraph.BuildConfigNode, lastSucces
...
 	}
 
+	var firstBuildToDisplay *buildgraph.BuildNode
+	firstTime := unversioned.Time{}
+	var secondBuildToDisplay *buildgraph.BuildNode
+	secondTime := unversioned.Time{}
+
 	// display the last successful build if specifically requested or we're going to display an active build for context
 	if lastSuccessfulBuild != nil && (includeSuccess || len(activeBuilds) > 0) {
-		out = append(out, describeBuildPhase(lastSuccessfulBuild.Build, &passTime, build.BuildConfig.Name, pushTargetResolved))
```
Comment (bparees) [inline pkg/cmd/cli/describe/projectstatus.go]: do you actually need to check if lastSuccessBuild!=nil?  seems like all this logic works fine even if "firstBuildToDisplay" or "secondBuildToDisplay" ends up getting assigned "nil"

**Comment by gabemontero** [comment]
The yum repo / mirror issues bit the test run.  I'll trigger a new one in a bit.

File: pkg/cmd/cli/describe/projectstatus.go (hunk 1 above)
Comment (gabemontero) [inline pkg/cmd/cli/describe/projectstatus.go]: Yep, that looks right on second glance ... I'll push an update in a bit.

Comment (gabemontero) [inline pkg/cmd/cli/describe/projectstatus.go]: update pushed

**Comment by gabemontero** [comment]
Cool, mirrors fixed and test passing.  Merge?

**Comment by bparees** [comment]
lgtm [merge]

**Pull request merged**